README.md

```diff
@@ -219,6 +220,14 @@ of the actual exit code.
 **Note:** This functionality was taken from Symfony's
 [Process](https://github.com/symfony/process) compoment.
 
+### Windows Compatibility
+
+Due to the blocking nature of `STDIN`/`STDOUT`/`STDERR` pipes on windows we cannot 
```

Comment:
Uhm, shouldn't that be _does not work on Windows_? IIRC `stream` does error out if it can't set the stream to non-blocking.